Components/Metrics/AdvancedMeanSquares/itkAdvancedMeanSquaresImageToImageMetric.hxx

```diff
@@ -671,42 +667,12 @@ AdvancedMeanSquaresImageToImageMetric<TFixedImage, TMovingImage>::AfterThreadedG
   value *= normal_sum;
 
   /** Accumulate derivatives. */
-  // compute single-threadedly
-  if (!Superclass::m_UseMultiThread && false) // force multi-threaded
```

Comment:
This `if (!x && false)` was also introduced with commit 2fc2c495837f64416263fa7c5d1e2bc4e3e061fa, "HUGE merge of the performance branch", November 14, 2013